en/controllers/middleware.rst

```diff
@@ -100,8 +125,8 @@ scripts, that add middleware::
 
     EventManager::instance()->on(
         'Server.buildMiddleware',
-        function ($event, $middlewareStack) {
-            $middlewareStack->add(new ContactPluginMiddleware());
+        function ($event, $middlewareQueueStack) {
```

Comment:
... this gave me some laughs... :)

So now it is sometimes middlewareQueue and sometimes middlewareQueueStack ;)?

*Personally I'd rename the core class to MiddlewareStack and alias it from middlewareQueue to be removed in 4.0. Stack is easier to type and speak.*